test/lib.bash

```diff
@@ -600,8 +600,8 @@ function check_serverless_alerts {
   monitoring_route=$(oc -n openshift-monitoring get routes alertmanager-main -oyaml -ojsonpath='{.spec.host}')
   # TODO(SRVKE-669) remove the filter for the pingsource-mt-adapter service once issue is fixed.
   curl -k -H "Authorization: Bearer $(oc -n openshift-monitoring create token prometheus-k8s)" \
-    "https://${monitoring_route}/api/v1/alerts" | \
-    jq -c '.data | map(select((.labels.service != "pingsource-mt-adapter") and (.labels.namespace == "'"${OPERATORS_NAMESPACE}"'" or .labels.namespace == "'"${EVENTING_NAMESPACE}"'" or .labels.namespace == "'"${SERVING_NAMESPACE}"'" or .labels.namespace == "'"${INGRESS_NAMESPACE}"'")))' > "${alerts_file}"
+    "https://${monitoring_route}/api/v2/alerts" | \
+    jq -c 'map(select((.labels.service != "pingsource-mt-adapter") and (.labels.namespace == "'"${OPERATORS_NAMESPACE}"'" or .labels.namespace == "'"${EVENTING_NAMESPACE}"'" or .labels.namespace == "'"${SERVING_NAMESPACE}"'" or .labels.namespace == "'"${INGRESS_NAMESPACE}"'")))' > "${alerts_file}"
 
   num_alerts=$(jq 'length' "${alerts_file}")
   num_apiremoved_alerts=$(jq '.[].labels.alertname=="APIRemovedInNextEUSReleaseInUse-quick"' "${alerts_file}" | wc -l)
```

Comment:
I know it's not what is being fixed here, this bit is weird...  it's not a select, so the output will be a sequence of `true|false` , so the number of lines will always be the same as the source, so the condition below will always be true?

Reply:
I think you're right.

Reply:
Pushed a fix.